pages/common/sudo.md

```diff
@@ -1,19 +1,19 @@
 # sudo
 
-> Execute a command as another user.
+> Executes a single command as another user.
```

Comment:
It can me multiple commands too right ? - `sudo cmdx && cmdy` ? Or does it run `cmdy` as the current user ?

Reply:
The `&&` would be another command and get executed as the current user, yeah. You need to use `sh -c '...'` or escape them IIRC.

Reply:
Ah right.